app/uk/gov/hmrc/agentsubscriptionfrontend/views/confirm_business.scala.html

```diff
@@ -18,20 +18,20 @@
 @import uk.gov.hmrc.agentsubscriptionfrontend.controllers.routes
 @import uk.gov.hmrc.agentsubscriptionfrontend.models.BusinessAddress
 @import uk.gov.hmrc.play.views.html.helpers._
-@import uk.gov.hmrc.agentsubscriptionfrontend.models.ConfirmYourAgency
+@import uk.gov.hmrc.agentsubscriptionfrontend.models.ConfirmBusiness
 
-@(confirmAgencyRadioForm: Form[ConfirmYourAgency], registrationName: String, utr: String, businessAddress: BusinessAddress)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(confirmBusinessRadioForm: Form[ConfirmBusiness], registrationName: String, utr: String, businessAddress: BusinessAddress)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
-@uk.gov.hmrc.agentsubscriptionfrontend.views.html.main_template(appConfig, title = error_prefix(confirmAgencyRadioForm) +  Messages("confirmYourAgency.title"), bodyClasses = None) {
+@uk.gov.hmrc.agentsubscriptionfrontend.views.html.main_template(appConfig, title = error_prefix(confirmBusinessRadioForm) +  Messages("confirmBusiness.title"), bodyClasses = None) {
 
-    <a href="@routes.CheckAgencyController.checkAgencyStatus(request.session.get("businessType"))" class="link-back">@Messages("button.back")</a>
+    <a href="@routes.BusinessIdentificationController.submitBusinessDetailsForm(request.session.get("businessType"))" class="link-back">@Messages("button.back")</a>
 
-    @if(confirmAgencyRadioForm.hasErrors) {
-        @errorSummary(Messages("error.summary.heading"), confirmAgencyRadioForm, Seq.empty, Some("check-agency-status"))
+    @if(confirmBusinessRadioForm.hasErrors) {
+        @errorSummary(Messages("error.summary.heading"), confirmBusinessRadioForm, Seq.empty, Some("check-business-status"))
```

Comment:
Should "check-business-details" be aligned with the page URL, i.e. "business-details"?